src/viam/ur/module/ur_arm_state.hpp

```diff
@@ -57,6 +57,7 @@ class URArm::state_ {
                                            std::ofstream arm_joint_positions_stream);
 
     bool is_moving() const;
+    std::string describe() const;
```

Comment:
Keep this method, even if we need to add others. It is likely to prove useful in the future.